src/Resources/config/rabbitmq_status_provider/guzzle.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="sonata.notification.rabbitmq.queue_status.guzzle_provider" class="Sonata\NotificationBundle\Service\RabbitMQQueueStatusGuzzleProvider">
+            <argument/>
```

Comment:
shall we close this like this?

```diff
- <service id="sonata.notification.rabbitmq.queue_status.guzzle_provider" class="Sonata\NotificationBundle\Service\RabbitMQQueueStatusGuzzleProvider">
# <service id="sonata.notification.rabbitmq.queue_status.guzzle_provider" class="Sonata\NotificationBundle\Service\RabbitMQQueueStatusGuzzleProvider"/>
```

Reply:
Ok. Because this service is not so important (BC), we can describe it shortly.